src/components/organisms/events/table.tsx

```diff
@@ -56,10 +64,22 @@ export const EventsTable = () => {
 			const event = sortedEvents[index];
 
 			return (
-				<EventRow event={event} key={key} onClick={() => navigate(`/events/${event.eventId}`)} style={style} />
+				<EventRow
+					event={event}
+					key={key}
+					onClick={() =>
+						navigate(
+							projectEventId
+								? `/projects/${projectId}/triggers/${triggerId}/events/${event.eventId}`
+								: `/events/${event.eventId}`
+						)
```

Comment:
Please extract to consts: onRowClick, sourceIdAddress, eventsAddress

Reply:
changed